tcms/testruns/tests/test_views.py

```diff
@@ -122,6 +122,30 @@ def test_show_create_new_run_page(self):
                 '<a href="%s">TC-%d: %s</a>' % (case_url, case.pk, case.summary),
                 html=True)
 
+    def test_should_be_able_to_create_new_run(self):
```

Comment:
:+1: for new scenario. Name should be `test_post_creates_new_run` to match changes requested above